packages/@ourworldindata/grapher/src/fullScreen/FullScreen.scss

```diff
@@ -0,0 +1,15 @@
+.FullScreenOverlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: $zindex-full-screen;
+    height: 100%;
+    width: 100%;
+
+    .FullScreenContent {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
```

Comment:
It centers the container horizontally and vertically. But this piece of code has since been [removed by Marcel](https://github.com/owid/owid-grapher/pull/2700) to fix an issue in Firefox.